### PR TITLE
chore(dtfs2-none): add env copy command

### PR DESCRIPTION
**Pull request by AlexBramhill**
## Introduction :pencil2:
added a command to copy env vars to all needed folders

### CODE REVIEW

File: utils/copy-env-vars.sh
Comment (avaitonis) [inline utils/copy-env-vars.sh:1]: For running cypress locally we might need .env in these folders too:
* e2e-tests/tfm
* e2e-tests/gef
* e2e-tests/portal
* e2e-tests/ukef

Comment (AlexBramhill) [reply utils/copy-env-vars.sh:2]: I don't have any of these set up and can run e2e tests fine? Does anyone else have/not have these please

Comment (avaitonis) [reply utils/copy-env-vars.sh:2]: I run e2e test in this way:
```
cd e2e-tests;
npx cypress open --project ./tfm;
```
What is your way?

Comment (avaitonis) [reply utils/copy-env-vars.sh:2]: .env in e2e-tests/tfm is required for my way

Comment (AlexBramhill) [reply utils/copy-env-vars.sh:2]: I do it the same... 
cd into e2e and `npx cypress open --project ./gef`...

I don't have the env vars in those folders. Odd. I'm happy to add if it helps you though

File: utils/copy-env-vars.sh
Comment (JoshBinns2000) [inline utils/copy-env-vars.sh:6]: Could we possibly have a section here for updating `MONGODB_URI` to use `localhost` instead of `dtfs-submissions-data` when we copy the `.env` file?

File: utils/copy-env-vars.sh
Comment (AlexBramhill) [reply utils/copy-env-vars.sh:2]: ah not ran recently with sql changes. adding

File: utils/copy-env-vars.sh
Comment (JoshBinns2000) [inline utils/copy-env-vars.sh:2]: I believe we now run all our utils commands from `utils`, so we should copy `.env` to `utils` instead of `utils/mock-data-loader`

Comment (AlexBramhill) [reply utils/copy-env-vars.sh:2]: what about npm run load? Why have we done this out of interest as it adds an extra step during setup/reloading

Comment (JoshBinns2000) [reply utils/copy-env-vars.sh:2]: `npm run load` runs from all of `.`, `./utils`, `./utils/mock-data-loader`. I think Abhi commonised our utils `package.json` so now all the utils can be run just from`./utils`

Comment (JoshBinns2000) [reply utils/copy-env-vars.sh:2]: What extra step has been added by making this change?

File: utils/copy-env-vars.sh
Comment (abhi-markan) [inline utils/copy-env-vars.sh:1]: Can we add `utils/**` to all `.dockerignore` please.

File: utils/copy-env-vars.sh
Comment (abhi-markan) [reply utils/copy-env-vars.sh:2]: Should not need `.env` inside all `e2e` just inside `e2e-tests` will suffice.
Please change:
* Remove `secrets`
* Update to `utils`
* Add `azure-functions/acbs-function`

File: utils/copy-env-vars.sh
Comment (abhi-markan) [reply utils/copy-env-vars.sh:6]: On the same though, can we also do the same for `SQL_DB_HOST`:
* `dtfs-sql` for root `.env`.
* `localhost` for all other `.envs`.

File: utils/copy-env-vars.sh
Comment (abhi-markan) [reply utils/copy-env-vars.sh:1]: Please use the same bash script pattern as `deploy.sh`.

**Review by abhi-markan — CHANGES_REQUESTED** [review]

File: utils/copy-env-vars.sh
Comment (AlexBramhill) [reply utils/copy-env-vars.sh]: this is the same :)

File: utils/copy-env-vars.sh
Comment (AlexBramhill) [reply utils/copy-env-vars.sh]: updated

File: utils/copy-env-vars.sh
Comment (AlexBramhill) [reply utils/copy-env-vars.sh]: misread what you meant by that -- thought you meant we as devs were meant to run in utils. updated

File: utils/copy-env-vars.sh
Comment (AlexBramhill) [reply utils/copy-env-vars.sh]: Also -- none of the files should be getting copied by our docker file currently

File: utils/copy-env-vars/.eslintrc.js
Comment (abhi-markan) [inline utils/copy-env-vars/.eslintrc.js:1]: Can this PR wait until my PR has merged as it will override few `.eslintrc.js` and `tsconfig` changes.

**Review by abhi-markan — CHANGES_REQUESTED** [review]

File: utils/copy-env-vars/.eslintrc.js
Comment (abhi-markan) [inline utils/copy-env-vars/.eslintrc.js:1]: Can we remove this file as root `.eslintrc.js` should be used in this instance rather than individual directories to have their own lint rules.

File: utils/copy-env-vars/copy-env-vars.ts
Comment (abhi-markan) [inline utils/copy-env-vars/copy-env-vars.ts:28]: Doc strings.

File: utils/copy-env-vars/copy-env-vars.ts
Comment (abhi-markan) [inline utils/copy-env-vars/copy-env-vars.ts:43]: Doc string

File: utils/copy-env-vars/copy-env-vars.ts
Comment (abhi-markan) [inline utils/copy-env-vars/copy-env-vars.ts:65]: Minor pedantic comment can you please add ✅ to keep all successful utils completion consistent. Thanks

File: utils/copy-env-vars/tsconfig.eslint.json
Comment (abhi-markan) [inline utils/copy-env-vars/tsconfig.eslint.json:1]: Please remove, should refer to root.

File: utils/copy-env-vars/tsconfig.json
Comment (abhi-markan) [inline utils/copy-env-vars/tsconfig.json:1]: Please remove, should refer to root.

File: utils/package.json
Comment (abhi-markan) [inline utils/package.json:11]: Can the script file name be `copy.ts` i.e. `ts-node copy-env-vars/copy.ts`

**Review by abhi-markan — CHANGES_REQUESTED** [review]

File: utils/copy-env-vars/copy-env-vars.ts
Comment (AlexBramhill) [reply utils/copy-env-vars/copy-env-vars.ts:43]: added -- we don't need types here as its ts and its inferred through this (and keeps it dry()

File: utils/copy-env-vars/index.ts
Comment (abhi-markan) [inline utils/copy-env-vars/index.ts:52]:
```suggestion
        console.info('.env file copied to %s', folderPathToCopyTo);
```

File: utils/copy-env-vars/index.ts
Comment (abhi-markan) [inline utils/copy-env-vars/index.ts:54]:
```suggestion
        console.error('Error while copying .env file to %s %o', folderPathToCopyTo, error);
```

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Good work, just two minor suggestions and can also please add Jira ticket number.

**Pull request closed without merging**
